spring-cloud-alibaba-starters/spring-cloud-starter-stream-rocketmq/src/main/java/com/alibaba/cloud/stream/binder/rocketmq/autoconfigurate/RocketMQBinderAutoConfiguration.java

```diff
@@ -65,4 +59,16 @@ public RocketMQMessageChannelBinder rocketMQMessageChannelBinder(
 				extendedBindingProperties, provisioningProvider);
 	}
 
+	@Configuration(proxyBeanMethods = false)
+	@ConditionalOnClass(HealthIndicator.class)
+	@ConditionalOnEnabledHealthIndicator("rocketmq")
+	static class KafkaBinderHealthIndicatorConfiguration {
```

Comment:
Need to modify the name from KafkaBinderHealthIndicatorConfiguration to RocketMQBinderHealthIndicatorConfiguration